test/consul_helpers.go

```diff
@@ -196,8 +196,7 @@ func checkEnterpriseInstall(t *testing.T, asgNameOutputVar string, terratestOpti
 		SshUserName: sshUser,
 		SshKeyPair:  keyPair.KeyPair,
 	}
-
-	maxRetries := 10
+	maxRetries := 20
```

Comment:
just curious - is there any derivation behind this value? I must be honest though - I'm lacking context, so excuse me if the question had been discussed before!

Reply:
Actually, the idea is to add more retries for the `Check Enterprise Install` which is failing on `circleci` like you can see in this [comment](https://github.com/hashicorp/terraform-aws-consul/pull/208#issuecomment-784114002).
BTW: we are trying to find a way to fix #209